### PR TITLE
Add Translation to Thai, Burmese, Swahili

### PR DESCRIPTION
**Pull request by lisandi**
And one addition to English Missing Correction Fluid

<sub>
By opening a pull request, I certify that I hold the intellectual property of the code I am submitting,
and I am granting the initial authors of WBO a perpetual, worldwide, non-exclusive, royalty-free, and irrevocable license to this code.
</sub>

### CODE REVIEW

File: server/translations.json [hunk 1]
```diff
@@ -77,6 +77,7 @@
     "text": "Text",
     "tools": "Tools",
     "view_source": "Source code on GitHub",
+    "white-out": "Correction fluid",
```
Comment (lovasoa) [inline server/translations.json:80]: Already defined on line 43

File: server/translations.json [hunk 2]
```diff
@@ -273,6 +274,46 @@
     "white-out": "修正液",
     "zoom": "拡大・縮小"
   },
+  "my": {
+    "board_name_placeholder": "ဝှိုက်ဘုတ်အမည် ... ",
+    "circle": "စက်ဝိုင်း",
+    "click_to_toggle": "toggle ကိုနှိပ်ပါ",
+    "click_to_zoom": "ချဲ့ရန်နှိပ်ပါ \ n shift key ကိုကိုင်ထားပြီးချဲ့ရန်နှိပ်ပါ",
```
Comment (lovasoa) [inline server/translations.json:281]:
```suggestion
    "click_to_zoom": "ချဲ့ရန်နှိပ်ပါ\nshift key ကိုကိုင်ထားပြီးချဲ့ရန်နှိပ်ပါ",
```

File: server/translations.json [hunk 3]
```diff
@@ -273,6 +274,46 @@
...
+    "loading": "တင်နေသည်",
+    "menu": "မီနူး",
+    "mousewheel": "ကြွက်ဘီး",
+    "mover": "ပြောင်း",
+    "named_private_board_description": "မင်းစိတ်ကြိုက် url တစ်ခုနဲ့ <strong> ကိုယ်ပိုင် whiteboard တစ်ခုကိုလည်းသင်ဖန်တီးနိုင်သည်။ နာမည်သိသူတိုင်း၎င်းကိုသုံးနိုင်သည်။ ",
+    "opacity": "အလင်းအမှောင်",
+    "open_public_board": "အများသုံး whiteboard သို့သွားပါ",
+    "pencil": "ဘောပင်",
+    "private_board_description": "လင့်ခ်တစ်ခုသာဖွင့်နိုင်သောကျပန်းအမည်ဖြင့် <b> ကိုယ်ပိုင် whiteboard </b> တစ်ခုကိုသင်ဖန်တီးနိုင်သည်။ ကိုယ်ပိုင်သတင်းအချက်အလက်များမျှဝေလိုလျှင်ဤအရာကိုသုံးပါ။,
+    "public_board_description": " အဆိုပါ <b> အများသုံး whiteboard </b> ကိုမည်သူမဆိုဖွင့်နိုင်ပါသည်။ ၎င်းသည်အမည်မသိသူစိမ်းများနှင့်သင်ဆေးသုတ်နိုင်သောပျော်ရွှင်စရာ၊ ရှုပ်ထွေးသောရှုပ်ထွေးမှုတစ်ခုဖြစ်သည်။,
```
Comment (lovasoa) [inline server/translations.json:303]:
```suggestion
    "public_board_description": "အဆိုပါ <b> အများသုံး whiteboard </b> ကိုမည်သူမဆိုဖွင့်နိုင်ပါသည်။ ၎င်းသည်အမည်မသိသူစိမ်းများနှင့်သင်ဆေးသုတ်နိုင်သောပျော်ရွှင်စရာ၊ ရှုပ်ထွေးသောရှုပ်ထွေးမှုတစ်ခုဖြစ်သည်။",
```

File: server/translations.json [hunk 4]
```diff
@@ -351,6 +392,86 @@
     "white-out": "Корректор",
     "zoom": "Лупа"
   },
+  "sw": {
+    "board_name_placeholder": "Jina la ubao mweupe ...",
+    "circle": "mduara",
+    "click_to_toggle": "Bonyeza kugeuza",
+    "click_to_zoom": "Bonyeza kuvuta ndani \ n Shikilia kitufe cha kuhama na ubofye ili kukuza",
```
Comment (lovasoa) [inline server/translations.json:399]:
```suggestion
    "click_to_zoom": "Bonyeza kuvuta ndani\nShikilia kitufe cha kuhama na ubofye ili kukuza",
```

File: server/translations.json [hunk 5]
```diff
@@ -273,6 +274,46 @@
...
+    "color": "အရောင်",
+    "configuration": "ဖွဲ့စည်းမှု",
+    "create_private_board": "ကိုယ်ပိုင် whiteboard တစ်ခုဖန်တီးပါ",
+    "ellipse": "ဘဲဥပုံ",
+    "eraser": "ခဲဖျက်",
+    "grid": "ဇယားကွက်",
+    "hand": "လက်",
+    "download": "ဒေါင်းလုပ်",
+    "index_title": "WBO မှကြိုဆိုပါသည်",
+    "introduction_paragraph": "WBO သည် <a href = \" https: //github.com/lovasoa/whitebophir \ "title = \" ဘီယာမပါ၊ လွတ်လပ်စွာပြောဆိုခွင့်၊ အခမဲ့ဖြစ်သည်။ ဤဆော့ဝဲကို AGPL လိုင်စင်အောက်တွင်ထုတ်ဝေသည်။ \ "> အခမဲ့နှင့်ပွင့်လင်းသောအရင်းအမြစ် </a> ပူးပေါင်းဆောင်ရွက်သည့်အွန်လိုင်း whiteboard ကိုအသုံးပြုသူများအား virtual whiteboard ကြီးတစ်ခုပေါ်တွင်တစ်ချိန်တည်းဆွဲခွင့်ပြုသည်။ whiteboard ကိုအသုံးပြုသူများအားလုံးအတွက်အချိန်နှင့်တပြေးညီအဆင့်မြှင့်တင်ခဲ့သည်။ ၎င်း၏အကြောင်းအရာကိုသိမ်းဆည်းထားသည်။ ၎င်းကိုအနုပညာ၊ ဖျော်ဖြေရေး၊ ဒီဇိုင်း၊ သင်ကြားမှုနှင့်သင်ကြားမှုကဲ့သို့သောအသုံးချမှုမျိုးစုံအတွက်အသုံးပြုနိုင်သည်",
```
Comment (lovasoa) [inline server/translations.json:292]:
```suggestion
    "introduction_paragraph": "WBO သည် <a href=\"https://github.com/lovasoa/whitebophir\" title=\"ဘီယာမပါ၊ လွတ်လပ်စွာပြောဆိုခွင့်၊ အခမဲ့ဖြစ်သည်။ ဤဆော့ဝဲကို AGPL လိုင်စင်အောက်တွင်ထုတ်ဝေသည်။\"> အခမဲ့နှင့်ပွင့်လင်းသောအရင်းအမြစ် </a> ပူးပေါင်းဆောင်ရွက်သည့်အွန်လိုင်း whiteboard ကိုအသုံးပြုသူများအား virtual whiteboard ကြီးတစ်ခုပေါ်တွင်တစ်ချိန်တည်းဆွဲခွင့်ပြုသည်။ whiteboard ကိုအသုံးပြုသူများအားလုံးအတွက်အချိန်နှင့်တပြေးညီအဆင့်မြှင့်တင်ခဲ့သည်။ ၎င်း၏အကြောင်းအရာကိုသိမ်းဆည်းထားသည်။ ၎င်းကိုအနုပညာ၊ ဖျော်ဖြေရေး၊ ဒီဇိုင်း၊ သင်ကြားမှုနှင့်သင်ကြားမှုကဲ့သို့သောအသုံးချမှုမျိုးစုံအတွက်အသုံးပြုနိုင်သည်",
```

File: server/translations.json [hunk 6]
```diff
@@ -351,6 +392,86 @@
...
+    "color": "rangi",
+    "configuration": "usanidi",
+    "create_private_board": "Unda ubao mweupe wa faragha",
+    "ellipse": "ellipse",
+    "eraser": "eraser",
+    "grid": "gridi",
+    "hand": "mkono",
+    "download": "Pakua",
+    "index_title": "Karibu kwenye WBO!",
+    "introduction_paragraph": "WBO ni <a href = \" https: //github.com/lovasoa/whitebophir \ "title = \" Huru kwa maana ya kusema bure, sio bia ya bure. Programu hii imechapishwa chini ya leseni ya AGPL. Yaliyomo yamehifadhiwa. Inaweza kutumika kwa matumizi anuwai, kama sanaa, burudani, kubuni, kufundisha na kufundisha. ",
```
Comment (lovasoa) [inline server/translations.json:410]:
```suggestion
    "introduction_paragraph": "WBO ni <a href=\"https: //github.com/lovasoa/whitebophir\" title=\"Huru kwa maana ya kusema bure, sio bia ya bure. Programu hii imechapishwa chini ya leseni ya AGPL. Yaliyomo yamehifadhiwa. Inaweza kutumika kwa matumizi anuwai, kama sanaa, burudani, kubuni, kufundisha na kufundisha.",
```

The `<a` tag opens and never closes in this translation

File: server/translations.json [hunk 7]
```diff
@@ -351,6 +392,86 @@
...
+    "tools": "zana",
+    "view_source": "Nambari ya chanzo kwenye GitHub",
+    "white-out": "maji ya kusahihisha",
+    "zoom": "Zoom"
+  },
+  "th": {
+    "board_name_placeholder": "ชื่อไวท์บอร์ด ...",
+    "circle": "วงกลม",
+    "click_to_toggle": "คลิกเพื่อสลับ",
+    "click_to_zoom": "คลิกเพื่อซูมเข้า \ nกดปุ่ม shift ค้างไว้แล้วคลิกเพื่อซูมออก",
```
Comment (lovasoa) [inline server/translations.json:439]:
```suggestion
    "click_to_zoom": "คลิกเพื่อซูมเข้า\nกดปุ่ม shift ค้างไว้แล้วคลิกเพื่อซูมออก",
```

File: server/translations.json [hunk 8]
```diff
@@ -351,6 +392,86 @@
...
+    "color": "สี",
+    "configuration": "การกำหนดค่า",
+    "create_private_board": "สร้างไวท์บอร์ดส่วนตัว",
+    "ellipse": "วงรี",
+    "eraser": "ยางลบ",
+    "grid": "กริด",
+    "hand": "มือ",
+    "download": "ดาวน์โหลด",
+    "index_title": "ยินดีต้อนรับสู่ WBO!",
+    "introduction_paragraph": "WBO เป็น <a href = \" https: //github.com/lovasoa/whitebophir \"title = \" ฟรีในแง่ของการพูดฟรี ไม่ใช่เบียร์ฟรี ซอฟต์แวร์นี้เผยแพร่ภายใต้ลิขสิทธิ์ AGPL \"> ฟรีและโอเพ่นซอร์ส </a> ไวท์บอร์ดออนไลน์สำหรับการทำงานร่วมกันที่อนุญาตให้ผู้ใช้จำนวนมากวาดบนไวท์บอร์ดเสมือนจริงขนาดใหญ่พร้อมกัน ไวท์บอร์ดได้รับการอัปเดตตามเวลาจริงสำหรับผู้ใช้ทั้งหมดและ เนื้อหาจะถูกบันทึกไว้ . สามารถใช้งานได้หลากหลายเช่นศิลปะ, บันเทิง, การออกแบบ, การสอนและการสอน ",
```
Comment (lovasoa) [inline server/translations.json:450]:
```suggestion
    "introduction_paragraph": "WBO เป็น <a href=\"https://github.com/lovasoa/whitebophir\" title=\"ฟรีในแง่ของการพูดฟรี ไม่ใช่เบียร์ฟรี ซอฟต์แวร์นี้เผยแพร่ภายใต้ลิขสิทธิ์ AGPL\">ฟรีและโอเพ่นซอร์ส</a> ไวท์บอร์ดออนไลน์สำหรับการทำงานร่วมกันที่อนุญาตให้ผู้ใช้จำนวนมากวาดบนไวท์บอร์ดเสมือนจริงขนาดใหญ่พร้อมกัน ไวท์บอร์ดได้รับการอัปเดตตามเวลาจริงสำหรับผู้ใช้ทั้งหมดและ เนื้อหาจะถูกบันทึกไว้ . สามารถใช้งานได้หลากหลายเช่นศิลปะ, บันเทิง, การออกแบบ, การสอนและการสอน ",
```

File: server/translations.json [hunk 9]
```diff
@@ -351,6 +392,86 @@
...
+    "loading": "กำลังโหลด",
+    "menu": "เมนู",
+    "mousewheel": "วงล้อเมาส์",
+    "mover": "กะ",
+    "named_private_board_description": "คุณยังสามารถสร้าง <strong> ไวท์บอร์ดส่วนตัวที่มีชื่อ </strong> ด้วย URL ที่กำหนดเองได้ ใครก็ตามที่รู้จักชื่อนี้สามารถเข้าถึงได้",
+    "opacity": "ความทึบ",
+    "open_public_board": "ไปที่ไวท์บอร์ดสาธารณะ",
+    "pencil": "ปากกา",
+    "private_board_description": "คุณสามารถสร้าง <b> ไวท์บอร์ดส่วนตัว </b> ด้วยชื่อแบบสุ่มที่สามารถเปิดได้ด้วยลิงก์เท่านั้น ใช้ตัวเลือกนี้หากคุณต้องการแบ่งปันข้อมูลส่วนตัว",
+    "public_board_description": " ใครๆ ก็เปิด <b> กระดานไวท์บอร์ดสาธารณะ </b> ได้ เป็นเรื่องที่น่ายินดีและไม่เป็นระเบียบ ซึ่งคุณสามารถวาดภาพร่วมกับคนแปลกหน้าที่ไม่ระบุชื่อได้ ทุกๆ อย่างในที่นี้เป็นเพียงชั่วคราวเท่านั้น",
```
Comment (lovasoa) [inline server/translations.json:461]:
```suggestion
    "private_board_description": "คุณสามารถสร้าง <b>ไวท์บอร์ดส่วนตัว</b> ด้วยชื่อแบบสุ่มที่สามารถเปิดได้ด้วยลิงก์เท่านั้น ใช้ตัวเลือกนี้หากคุณต้องการแบ่งปันข้อมูลส่วนตัว",
    "public_board_description": "ใครๆ ก็เปิด <b>กระดานไวท์บอร์ดสาธารณะ</b> ได้ เป็นเรื่องที่น่ายินดีและไม่เป็นระเบียบ ซึ่งคุณสามารถวาดภาพร่วมกับคนแปลกหน้าที่ไม่ระบุชื่อได้ ทุกๆ อย่างในที่นี้เป็นเพียงชั่วคราวเท่านั้น",
```

File: server/translations.json [hunk 10]
```diff
@@ -351,6 +392,86 @@
...
+    "hand": "มือ",
+    "download": "ดาวน์โหลด",
+    "index_title": "ยินดีต้อนรับสู่ WBO!",
+    "introduction_paragraph": "WBO เป็น <a href = \" https: //github.com/lovasoa/whitebophir \"title = \" ฟรีในแง่ของการพูดฟรี ไม่ใช่เบียร์ฟรี ซอฟต์แวร์นี้เผยแพร่ภายใต้ลิขสิทธิ์ AGPL \"> ฟรีและโอเพ่นซอร์ส </a> ไวท์บอร์ดออนไลน์สำหรับการทำงานร่วมกันที่อนุญาตให้ผู้ใช้จำนวนมากวาดบนไวท์บอร์ดเสมือนจริงขนาดใหญ่พร้อมกัน ไวท์บอร์ดได้รับการอัปเดตตามเวลาจริงสำหรับผู้ใช้ทั้งหมดและ เนื้อหาจะถูกบันทึกไว้ . สามารถใช้งานได้หลากหลายเช่นศิลปะ, บันเทิง, การออกแบบ, การสอนและการสอน ",
+    "keyboard_shortcut": "แป้นพิมพ์ลัด",
+    "loading": "กำลังโหลด",
+    "menu": "เมนู",
+    "mousewheel": "วงล้อเมาส์",
+    "mover": "กะ",
+    "named_private_board_description": "คุณยังสามารถสร้าง <strong> ไวท์บอร์ดส่วนตัวที่มีชื่อ </strong> ด้วย URL ที่กำหนดเองได้ ใครก็ตามที่รู้จักชื่อนี้สามารถเข้าถึงได้",
```
Comment (lovasoa) [inline server/translations.json:456]:
```suggestion
    "named_private_board_description": "คุณยังสามารถสร้าง <strong>ไวท์บอร์ดส่วนตัวที่มีชื่อ</strong> ด้วย URL ที่กำหนดเองได้ ใครก็ตามที่รู้จักชื่อนี้สามารถเข้าถึงได้",
```

File: server/translations.json [hunk 11]
```diff
@@ -273,6 +274,46 @@
...
+    "hand": "လက်",
+    "download": "ဒေါင်းလုပ်",
+    "index_title": "WBO မှကြိုဆိုပါသည်",
+    "introduction_paragraph": "WBO သည် <a href = \" https: //github.com/lovasoa/whitebophir \ "title = \" ဘီယာမပါ၊ လွတ်လပ်စွာပြောဆိုခွင့်၊ အခမဲ့ဖြစ်သည်။ ဤဆော့ဝဲကို AGPL လိုင်စင်အောက်တွင်ထုတ်ဝေသည်။ \ "> အခမဲ့နှင့်ပွင့်လင်းသောအရင်းအမြစ် </a> ပူးပေါင်းဆောင်ရွက်သည့်အွန်လိုင်း whiteboard ကိုအသုံးပြုသူများအား virtual whiteboard ကြီးတစ်ခုပေါ်တွင်တစ်ချိန်တည်းဆွဲခွင့်ပြုသည်။ whiteboard ကိုအသုံးပြုသူများအားလုံးအတွက်အချိန်နှင့်တပြေးညီအဆင့်မြှင့်တင်ခဲ့သည်။ ၎င်း၏အကြောင်းအရာကိုသိမ်းဆည်းထားသည်။ ၎င်းကိုအနုပညာ၊ ဖျော်ဖြေရေး၊ ဒီဇိုင်း၊ သင်ကြားမှုနှင့်သင်ကြားမှုကဲ့သို့သောအသုံးချမှုမျိုးစုံအတွက်အသုံးပြုနိုင်သည်",
+    "keyboard_shortcut": "ကီးဘုတ်ဖြတ်လမ်း",
+    "loading": "တင်နေသည်",
+    "menu": "မီနူး",
+    "mousewheel": "ကြွက်ဘီး",
+    "mover": "ပြောင်း",
+    "named_private_board_description": "မင်းစိတ်ကြိုက် url တစ်ခုနဲ့ <strong> ကိုယ်ပိုင် whiteboard တစ်ခုကိုလည်းသင်ဖန်တီးနိုင်သည်။ နာမည်သိသူတိုင်း၎င်းကိုသုံးနိုင်သည်။ ",
```
Comment (lovasoa) [inline server/translations.json:298]: There is a `</strong>` missing here

File: server/translations.json [hunk 12]
```diff
@@ -351,6 +392,86 @@
...
+    "loading": "Inapakia",
+    "menu": "menyu",
+    "mousewheel": "gurudumu la panya",
+    "mover": "kuhama",
+    "named_private_board_description": "Unaweza pia kuunda <strong> ubao mweupe wa faragha wenye jina </strong> na url ya kawaida. Mtu yeyote anayejua jina anaweza kuipata.",
+    "opacity": "Opacity",
+    "open_public_board": "Nenda kwenye ubao mweupe wa umma",
+    "pencil": "kalamu",
+    "private_board_description": "Unaweza kuunda <b> ubao mweupe wa faragha </b> na jina la nasibu ambalo linaweza kufunguliwa tu na kiunga. Tumia hii ikiwa unataka kushiriki habari za kibinafsi.",
+    "public_board_description": " Bodi nyeupe ya umma </b> inaweza kufunguliwa na mtu yeyote. Ni fujo yenye furaha, isiyo na mpangilio ambapo unaweza kupaka rangi na wageni wasiojulikana. Kila kitu hapo ni cha muda mfupi.",
```
Comment (lovasoa) [inline server/translations.json:421]: There is a `</b>` without a `<b>` here

**Review by lovasoa — CHANGES_REQUESTED** [review]
Awesome, thank you very much for this contribution ! Do you speak all of these languages natively ?

I found a few problems with the HTML tags

**Comment by lisandi** [comment]
Hi Ophir LOJKINE
Unfortunately not all those languages even I would wish too but I am good
at motivating others to help me :-)
I run a company in Thailand and work to make All Inclusive Montessori
Education for All a reality and that involves activities in Africa
(Tanzania, Uganda, Ruanda, and DRC for now) and so I practice the language
skills but still far from perfect. But as my father who was a great leader
in a company too once said after a fire destroyed the work of years which
now had to be rebuilt in less than one year to meet the deadlines and he
worked through all day - no holidays - from early morning to very late and
even on Saturday Sunday- simply working through for nearly one year. I
asked him why he did that and he said to me that if he as a leader would
have quitted with sickness or "I can no more" all others would have
considered that too and the project would never have met the deadline. So
he had to give THE example that it is worth giving really all to meet the
goal. I try to follow that advice and my goal is "All Inclusive Montessori
Education for All" and to meet that goal we need to motivate people not
only to donate time and funds but also themselves - and often also their
peers - to make that happen. In the case of the translations, it is simply
necessary to have them as otherwise, the people won't understand and then
not use the online tools - But actually, it will be the provided
online tools which will help us to reach the goals and actually to make it
possible thanks worldwide cooperation tools. In the rich world, we have
tons of expensive private Montessori Schools and in the poor world we have
none of them - well also some pretty expensive ones - but what is needed is
FREE Montessori Education for All - so we need to train teachers at first
place - but to become a Montessori teacher is incredible expensive too as
the name "Montessori" sells pretty good. I like to provide FREE resources -
FREE Training and of course a highly qualification certificate which is
affordable for every teacher who is able to also finance a smartphone.
Thanks to tools like WBO we have now tools to make that happen! ;-) - and
we use it also on other sites too i.e. for Churches and Communities - as
those will hopefully help us too by spreading the word, raising funds, and
motivating others.

And the Burmese - well my grandson is going to a Good Shepherd School here
in Phuket/Thailand which takes care of Burmese Migrant Children. He is
officially Homeschooling but going there for socializing and of course, for
learning - even he is 100%, Thai. But I decided to send him there because
that school knows how to deal with his ADHD Problems pretty well and they
don't beat the children - like they still do in many Thai schools. So here
are my Burmese Helpers ;-) beside my Thai Staff in the company itself. The
Swahili was helped by 2 people who cross-checked it from Tanzania and
Uganda.

Yeah with the HTML I actually had a bit of a problem as we edited as
described directly on Github which was probably not a good idea and then we
saved it and saw in the green areas diffs that there might be problems but
could not find them - That is my problem with especially Thai and Burmese
as here the letters sometimes get written from left to right but also from
right to left in some cases - i.e. My wife is called Urai but when you
would read the Thai Letters letter by better like you are used to in
western world then you would read Uair with the r at the end! - อุไร - the
first bug letter here has actually NO sound and gets used when a word
starts with a Vowel. The "U" is that tiny little Hook below that big first
letter. - The second letter is the "ai" (pronounced like a German "ai" "a
meted to i") and the last letter is the "r". For me it is always a headache
to find out the words in Thai and Burmese as in both languages they have no
separation of words either ;-) I, therefore, prefer languages like Russian
or Swahili where you can separate the words and that makes it much easier
to understand and learn them. - The process in our company is that I
usually start a translation to a language and then ask them to help me to
get it actually done ;-) completely - while I sit often hours to find and
write the first part.e. 10% and they do the rest 90% in no time at all :-)
But without me starting they would never do it at all! - Well and often
they then correct me of course pointing out the writing errors in my 10%
hahaha. Here in Phuket I can find nearly all languages of the world -
really a great place and even in our company we are very multilingual too.

Keep up your good work as it is doing and enabling great things to come.

Thanks a lot!
tools

With kind regards,
*
<https://about.me/andibecker?promo=email_sig&utm_source=product&utm_medium=email_sig&utm_campaign=edit_panel&utm_content=thumb>
Andreas Becker about.me/andibecker
<https://about.me/andibecker?promo=email_sig&utm_source=product&utm_medium=email_sig&utm_campaign=edit_panel&utm_content=thumb>
**LisAndi Co. Ltd.*, Phuket, Thailand (lisandi.com)
15/21 M.2 Viset Road, Rawai, Muang, Phuket, Thailand 83130

Phone: +66 (0)76 530 595
GTalk/Facetime/eMail:  ***@***.***
Mobile/WhatsApp/Line: +66 (0)81 606 3378
Facebook/Line/Messenger: andibecker
Skype: lisandi
--------------------------------------------------

**Comment by lovasoa** [comment]
It sounds like you are doing a lot of good work, congratulations.

For the translations, I tried to fix the markup myself, but it's hard to do so without understanding the text myself.

I'll merge this for now, but can you please have the native speakers check the result on the live website, and open new pull requests if anything seems wrong ?

**Pull request merged**